### ISSUE TITLE
Cannot null a datetime

### ISSUE THREAD

**Issue by theraccoonbear** [EarthlingInteractive/PHPDBC#1]
I'm trying to set a datetime column to null.  Running a manual query with Sequel Pro allows me to set the column to null, but passing a PHP null as a named parameter to the doQuery() method is setting the column to 0000-00-00 00:00:00.

**Comment by TOGoS** [comment]
Can you paste the output of `$sqlRunner->quoteParams( $sql, $params )`?

**Comment by theraccoonbear** [comment]
```
INSERT INTO referrals (id, created, referred_by_archive_timestamp) VALUES ('183', '2016-03-16 11:56:10', '')

ON DUPLICATE KEY UPDATE
modified = '2016-03-16 11:56:10',
referred_by_archive_timestamp = ''
```

**Comment by TOGoS** [comment]
Sorry, should have asked this earlier, but can you also var_export your $sql and $params and get_class($sqlRunner)?  Then I can make a unit test.

**Comment by theraccoonbear** [comment]
In order this is: get_class(), $sql, $params, and the output of quoteParams()

```
EarthIT_DBC_PDOSQLRunner
----
INSERT INTO referrals (id, created, referred_by_archive_timestamp) VALUES ({id}, {created}, {referred_by_archive_timestamp})

ON DUPLICATE KEY UPDATE
modified = {modified},
referred_by_archive_timestamp = {referred_by_archive_timestamp}
----
array(4) {
  ["id"]=>
  string(3) "183"
  ["created"]=>
  string(19) "2016-03-16 12:06:30"
  ["referred_by_archive_timestamp"]=>
  NULL
  ["modified"]=>
  string(19) "2016-03-16 12:06:30"
}
----
INSERT INTO referrals (id, created, referred_by_archive_timestamp) VALUES ('183', '2016-03-16 12:04:56', '')

ON DUPLICATE KEY UPDATE
modified = '2016-03-16 12:04:56',
referred_by_archive_timestamp = ''
```

**Comment by TOGoS** [comment]
I'd been putting too much faith in PDO#quote.

I think this is fixed by version 1.5.1: f66914318cdbaf13445cea5edce5de65f3c4e474
